docs/topics/wildfly-swarm/docs/reference/fractions/elytron.adoc

```diff
@@ -0,0 +1,996 @@
+To request (but not to require) a client certificate on SSL handshake. If a security domain is referenced and supports X509 evidence, this will be set to true automatically. Ignored when need-client-auth is set.
+
+swarm.elytron.server-ssl-contexts._KEY_.wrap:: 
+Should the SSLEngine, SSLSocket, and SSLServerSocket instances returned be wrapped to protect against further modification.
+
+swarm.elytron.service-loader-http-server-mechanism-factories._KEY_.available-mechanisms:: 
+The HTTP mechanisms available from this factory instance.
+
+swarm.elytron.service-loader-http-server-mechanism-factories._KEY_.module:: 
+The module to use to obtain the classloader to load the factories, if not specified the classloader to load the resource will be used instead.
+
+swarm.elytron.service-loader-sasl-server-factories._KEY_.available-mechanisms:: 
+The SASL mechanisms available from this factory after all filtering has been applied.
+
+swarm.elytron.service-loader-sasl-server-factories._KEY_.module:: 
+The module to use to obtain the classloader to load the factories, if not specified the classloader to load the resource will be used instead.
+
+swarm.elytron.simple-permission-mappers._KEY_.mapping-mode:: 
+The mapping mode that should be used in the event of multiple matches.
+
+swarm.elytron.simple-permission-mappers._KEY_.permission-mappings:: 
+The defined permission mappings.
+
+swarm.elytron.simple-regex-realm-mappers._KEY_.delegate-realm-mapper:: 
+The RealmMapper to delegate to if there is no match using the pattern.
+
+swarm.elytron.simple-regex-realm-mappers._KEY_.pattern:: 
+The regular expression which must contain at least one capture group to extract the realm from the name. If the regular expression matches more than one capture group, the first capture group is used.
+
+swarm.elytron.simple-role-decoders._KEY_.attribute:: 
+The name of the attribute from the identity to map directly to roles.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.attribute-synchronized:: 
+Whether every event should be immediately synchronised to disk.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.format:: 
+The format to use to record the audit event.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.max-backup-index:: 
+The maximum number of files to backup when rotating.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.path:: 
+Path of the file to be written.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.relative-to:: 
+The relative path to the audit log.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.rotate-on-boot:: 
+Whether the file should be rotated before the a new file is set.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.rotate-size:: 
+The log file size the file should rotate at.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.suffix:: 
+Format of date used as suffix of log file names in java.time.format.DateTimeFormatter. The suffix does not play a role in determining when the file should be rotated.
```

Comment:
add formatting to java classpath

Reply:
Note to self: not easily fixable in our sources.